qa/rpc-tests/llmq-is-spark.py

```
@@ -31,8 +31,10 @@ def run_test(self):
         self.nodes[0].generate(501 - self.nodes[0].getblockcount())
 
         sparkaddress = self.nodes[0].getnewsparkaddress()[0]
+        # Mint a few larger single coins so spendspark can cover amount+fee from
+        # one coin (multi-input Spark spends are disabled).
         for i in range(0, 3):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)